staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/compile_test.go

```diff
@@ -49,6 +49,11 @@ func TestCompileValidatingPolicyExpression(t *testing.T) {
 			expressions: []string{"object.foo < params.x"},
 			hasParams:   true,
 		},
+		{
```

Comment:
Please also add integration tests for this change. I would expect at least one integration test for namespace scoped and one for cluster scoped.

Reply:
not in `compile_test.go` though. The expressions are not evaluated so whether namespace exists does not matter.